inginious/frontend/templates/task.html

```diff
@@ -87,20 +85,6 @@ <h3>{{ _("Information") }}</h3>
                 {% endif %}
             </td>
         </tr>
-        {% set ns = namespace(tags="") %}
-        {% for category in task_categories %}
-            {% set tag = course_tags[category] %}
```

Comment:
We can keep this piece of code for categories only (as it's probably done in ``inginious/frontend/templates/course_admin/submission.html``).